CSCAutomateUnitTest/CloudSkillApiTests.cs

```diff
@@ -29,10 +29,10 @@ public async Task AddLearnerAsyncTest()
             Configuration config = await TestHelper.GetConfigurationAsync();
             CloudSkillApi cscApi = new CloudSkillApi(config);

```

Comment:
Ideally, the API calls should be mocked out and not actually hit real services. Instead, we should put the APIs behind an interface and back that interface up with a "Test Instance."

I can work on that if you want.